lunch-application/src/theme/AppHeader.vue

```diff
@@ -2,8 +2,8 @@
   <nav class="nav has-shadow">
     <div class="container">
       <router-link to="/Home" exact>
-        <img src="http://bit.ly/vue-img"
-          alt="Lunch App" />
+        <!-- <img src="../assets/chi_logo-trans.png" -->
+          <!-- alt="Lunch App" /> -->
```

Comment:
NB: this is commented out because the app doesn't have a png loader specified yet.